<launch>

   <arg name="domain" />
   <arg name="problem" />

   <node pkg="tfd_modules" type="tfd_plan" name="tfd_plan" args="$(arg domain) $(arg problem)" required="true">
      <rosparam command="load" file="$(find tfd_modules)/config/tfd_modules_params.yaml" />
   </node>

</launch>
